sdks/java/maven-archetypes/java8-examples/src/main/resources/archetype-resources/src/main/java/DebuggingWordCount.java

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
This is the third copy of the wordcount pipelines; we should try to do something better.